neuralmonkey/runners/perplexity_runner.py

```diff
@@ -5,7 +5,7 @@
 computes its perplexities given the decoder.
 """
 
-# tests: lint
+# tests: lint, mypy
```

Comment:
Jako nejsnadnější se mi jeví to tvoje rebasnout na decoder_refactor a tam ty změny udělat, pokud nechceš čekat, až se decoder_refactor mergne. Jiný PR s tim podle mě konflikty mít nebudou.